### DISCUSSION TITLE
想给CopilotView的Log也加个HH:mm:ss

### DISCUSSION THREAD

**Discussion by status102** [MaaAssistantArknights/MaaAssistantArknights#7400] [Ideas]
如果参照TaskQueue那边的话，提示内容也会有time，就很抽象
要不把提示摘出去

![image](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/102887808/286b5972-3a04-45a9-b07e-1fa27da33e45)

**Comment by ABA2396** [comment]
给LogItemViewModel加个参数是不是就好了）

**Reply by status102** [reply]
好想法
